addons/sale/controllers/product_configurator.py

```diff
@@ -44,7 +44,14 @@ def get_combination_info(self, product_template_id, product_id, combination, add
         ProductTemplate = request.env['product.template']
         if 'context' in kw:
             ProductTemplate = ProductTemplate.with_context(**kw.get('context'))
-        return ProductTemplate.browse(int(product_template_id))._get_combination_info(combination, int(product_id or 0), int(add_qty or 1), pricelist)
+        product_template = ProductTemplate.browse(int(product_template_id))
+        res = product_template._get_combination_info(combination, int(product_id or 0), int(add_qty or 1), pricelist)
+        if 'parent_combination' in kw:
```

Comment:
2 small remarks here:

- Why the condition on 'parent_combination' ?
It works because it's passed as "[]" for a product that has no parent_combination but it's a little weird.

- Maybe the "is_combination_possible" key should be included in the base "_get_combination_info" method?

Reply:
- You understood that correctly. It was just to avoid adding one more key to the rpc since it wouldn't carry any additional information.

- I considered putting it on `_get_combination_info` but ultimately I decided to make the diff minimal and not have to change the signature/return of the method too. This can be justified because here is the only place where this is needed, all other calls of the method don't need to compute this.

For those 2 points I'm fine either way, as per reviewer will think is best.